From 0db82219bb2ad10dcdf3ea8fe8e226858190c83b Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Thu, 22 Mar 2018 01:47:07 +0200
Subject: [PATCH] Fix "left handed" property not set on all pointers

Remove conditional that prevents the LIBINPUT_PROP_LEFT_HANDED{,DEFAULT}
properties from being set on all pointer devices (only the first got it).
This appears to be a debugging left-over accidentally merged in
6d3bd4544a6a2f194fa.

https://bugs.freedesktop.org/show_bug.cgi?id=105667

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/xf86libinput.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/xf86libinput.c b/src/xf86libinput.c
index 83ab75d..f9ed8ba 100644
--- a/src/xf86libinput.c
+++ b/src/xf86libinput.c
@@ -4930,9 +4930,6 @@ LibinputInitLeftHandedProperty(DeviceIntPtr dev,
 	if (!subdevice_has_capabilities(dev, CAP_POINTER|CAP_TABLET))
 		return;
 
-	if (prop_left_handed != 0)
-		return;
-
 	if (!libinput_device_config_left_handed_is_available(device) ||
 	    driver_data->capabilities & CAP_TABLET)
 		return;
-- 
2.16.2

